Expecto.Tests/Tests.fs

```diff
@@ -1396,57 +1396,61 @@ let taskTests =
         fun ms -> task { return ms.CanWrite ==? true }
     ]
 
-    testCaseTask "simple" <| task {
```

Comment:
I didn't change any of the tests, just grouped them with testList to fix the name conflict